src/main/java/com/greghaskins/spectrum/internal/Suite.java

```diff
@@ -126,7 +126,8 @@ public void addHook(final HookContext hook) {
   }
 
   private Hooks getHooksFor(final Child child) {
-    Hooks allHooks = this.parent.getInheritableHooks().plus(this.hooks);
+    Hooks allHooks = isAtomic() ? new Hooks() : this.parent.getInheritableHooks();
```

Comment:
This is the thing which stops an _atomic_ parent conveying its own inherited hooks down to a child.